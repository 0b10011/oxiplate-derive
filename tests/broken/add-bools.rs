use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{{ a + b }}"]
struct Data {
    a: bool,
    b: bool,
}

fn main() {
    let data = Data { a: true, b: false };

    print!("{}", data);
}
