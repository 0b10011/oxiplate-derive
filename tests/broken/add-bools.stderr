error[E0369]: cannot add `bool` to `bool`
 --> tests/broken/add-bools.rs:4:27
  |
4 | #[oxiplate_inline = "{{ a + b }}"]
  |                         - ^ - bool
  |                         |
  |                         bool
